app/src/pages/inside/stepPage/modals/makeDecisionModal/executionSection/executionSection.jsx

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { Fragment, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { useIntl } from 'react-intl';
+import { useDispatch, useSelector } from 'react-redux';
+import { activeProjectSelector } from 'controllers/user';
+import { activeFilterSelector } from 'controllers/filter';
+import { fetch } from 'common/utils';
+import { URLS } from 'common/urls';
+import { NOTIFICATION_TYPES, showNotification } from 'controllers/notification';
+import { TestItemDetails } from 'pages/inside/stepPage/modals/makeDecisionModal/elements/testItemDetails';
+import { historyItemsSelector } from 'controllers/log';
+import { TO_INVESTIGATE_LOCATOR_PREFIX } from 'common/constants/defectTypes';
+import {
+  ALL_LOADED_TI_FROM_HISTORY_LINE,
+  CURRENT_EXECUTION_ONLY,
+  SEARCH_MODES,
+  SHOW_LOGS_BY_DEFAULT,
+} from '../constants';
+import { OptionsSection } from './optionsSection';
+import styles from './executionSection.scss';
+import { messages } from '../messages';
+
+const cx = classNames.bind(styles);
+
+export const ExecutionSection = ({
+  modalState,
+  setModalState,
+  isNarrowView,
+  isBulkOperation,
+  eventsInfo,
+}) => {
+  const { formatMessage } = useIntl();
+  const dispatch = useDispatch();
+  const activeProject = useSelector(activeProjectSelector);
+  const activeFilter = useSelector(activeFilterSelector);
+  const [currentItemsLoading, setCurrentItemsLoading] = useState(false);
+  const [loading, setLoading] = useState(false);
+  const { optionValue, currentTestItems } = modalState;
+  const historyItems = useSelector(historyItemsSelector).filter(
+    (item) =>
+      !(
+        (item.issue && !item.issue.issueType.startsWith(TO_INVESTIGATE_LOCATOR_PREFIX)) ||
+        item.id === currentTestItems[0].id
+      ),
```

Comment:
The [error](https://github.com/reportportal/service-ui/pull/2908#pullrequestreview-805388512) probably come from here.